operators/pkg/info/info.go

```diff
@@ -0,0 +1,102 @@
+// Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+// or more contributor license agreements. Licensed under the Elastic License;
+// you may not use this file except in compliance with the Elastic License.
+
+package info
+
+import (
+	"github.com/elastic/cloud-on-k8s/operators/pkg/controller/common/reconciler"
+	"github.com/elastic/cloud-on-k8s/operators/pkg/utils/k8s"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apimachinery/pkg/types"
+	"k8s.io/client-go/kubernetes"
+	"k8s.io/client-go/rest"
+)
+
+const UUIDCfgMapName = "elastic-operator-uid"
+
+var (
+	// Variables that must be set during the manager initialization by calling `Setup(ns, cfg)`.
+	operatorNamespace string
+	k8sDistribution   string
+
+	// Operator UUID set through a config map reconciliation.
+	operatorUUID types.UID
```

Comment:
I'm 👎 on having globals, especially globals that cannot be initialized through `init()`, which seems to be the case here. Can we work out a way to get rid of them?

Reply:
Although it may seem simpler, I'm agree that having global state is not good. It reminds me this [tl;dr: magic is bad; global state is magic → no package level vars; no func init](https://peter.bourgon.org/blog/2017/06/09/theory-of-modern-go.html).
I will do the small effort to create an instance of `Info` in the main and propagate it in the `kibana.ReconcileKibana`.